.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug report
```

Comment:
why is this a yml file and not an md file?

Reply:
New feature
It's a form instead of a text block.

see here https://github.com/eseiler/sharg-parser/issues/new/choose

Reply:
Nice!!